perceval/components/unitary_components.py

```diff
@@ -415,6 +415,23 @@ def apply(self, r, sv):
         })
         return nsv
 
+    def break_in_2_mode_perms(self):
+
+        perm_vec_req = self.perm_vector
+        perm_len = len(perm_vec_req)
```

Comment:
Technically calling the method `break_in_2_mode_perms()` would throw a different kind of error. Since the method returns a circuit, what the user has is not a `PERM` anymore. The method is defined in `PERM`  and not a `Circuit` so there would be an error.

Yes, I was planning to have a method like that in MPS, but you are correct - it would be more useful to have it in `Circuit`.